docs/administration.md

```diff
@@ -92,7 +92,41 @@ The Pilosa server should support PQL versioning using HTTP headers. On each requ
 
 #### Upgrading
 
-When upgrading, upgrade clients first, followed by server for all Minor and Patch level changes.
+To upgrade Pilosa:
+
+1. First, upgrade the [client libraries](../client-libraries/) you are using in your application. Generally, a client version `X` will be compatible with the Pilosa server version `X` and earlier. For example, `python-pilosa 0.9.0` is compatible with both `pilosa 0.8.0` and `pilosa 0.9.0`.
+2. Next, download the latest release from our [installation page](/docs/latest/installation/) or from the [release page on Github](https://github.com/pilosa/pilosa/releases).
+3. Shut down the Pilosa cluster. A graceful shutdown should ensure that all data files are flushed to disk.
```

Comment:
how is a graceful shutdown accomplished?

Reply:
I could probably leave out the sentence "A graceful shutdown..." as it can be assumed that "shut down the pilosa cluster" would entail a graceful shutdown. A user has to "try" to kill pilosa non-gracefully (ie `kill -9` or pull the power plug).